src/sql/parts/dashboard/contents/controlHostContent.component.ts

```diff
@@ -73,4 +72,13 @@ export class ControlHostContent implements OnInit {
 	public get controlType(): string {
 		return this._type;
 	}
+
+	public get agentRefresh(): boolean {
```

Comment:
same comment as above.  I think agent methods should be in the child components and this component should implement generic methods, like `refresh` for example.